build/build_android_llm_demo.sh

```diff
@@ -37,6 +37,7 @@ build_android_native_library() {
   cmake . -DCMAKE_INSTALL_PREFIX="${CMAKE_OUT}" \
     -DCMAKE_TOOLCHAIN_FILE="${ANDROID_NDK}/build/cmake/android.toolchain.cmake" \
     -DANDROID_ABI="${ANDROID_ABI}" \
+    -DANDROID_PLATFORM=android-26 \
```

Comment:
I thought we bump to more recent version?

Reply:
Seems that here 26 is needed?  Thought 30 is stricter.

Reply:
26 is needed here since otherwise I get errors in the build like attached 
![image](https://github.com/user-attachments/assets/8ad31502-ce68-4fdf-be78-181b77147bd5)